### ISSUE TITLE
Does this software run on the NanoPi Neo?

### ISSUE THREAD

**Issue by faddat** [universelabs/universe-storage#8]
We need to validate weather or not we can run this on the NanoPi Duo.  

Does it have enough memory to deal with Go?

**Comment by faddat** [comment]
my early indications on the nanopi duo aren't so great.  While it can run much more software than the esp32/feather, it's pretty atrocious to get running.  In fact I've played with SBCs for years, and I'm not yet in this one (SSH, I mean)

That said, I'd love to find some kind of happy medium.  But I'm not sure that it exists.  If for example I made a ROM for this that disabled wifi, setup bluetooth..... it'll be a journey no matter which device we use, I guess I should be saying. (edited) 

I'm going to change tactics a little and look at what Armbian has for this, and maybe go through the Armbian supported devices list.

Armbian is a linux distribution that's really more of a giant, image-building bash script.

Armbian mentions that this device has such a poor wifi chipset that we should expect issues.  I'm thinking to move over to another device after this because of that.  Possibly the nanopi air 2 (which I have as well)

https://www.armbian.com/nanopi-duo/
